ui/app/AppLayouts/Chat/popups/community/BackUpCommunityPopup.qml

```diff
@@ -0,0 +1,150 @@
+            text: qsTr("You should keep it safe and only share it with people you trust to take ownership of your community")
+            wrapMode: Text.WordWrap
+            font.pixelSize: 13
+            color: Theme.palette.baseColor1
+        }
+
+        StatusBaseText {
+            Layout.fillWidth: true
+            Layout.preferredHeight: 34
+            text: qsTr("You can also use this key to import your community on another device")
+            wrapMode: Text.WordWrap
+            font.pixelSize: 13
+            color: Theme.palette.baseColor1
+        }
+        },
+        Item {
+            TextEdit {
+                id: validationInput
+                property string previousText: text
+                width: parent.width
+                height: 24
+                anchors.centerIn: parent
+                anchors.verticalCenterOffset: -indicationText.height
+                horizontalAlignment: Text.AlignHCenter
+                color: Theme.palette.baseColor1
+                cursorDelegate: Rectangle {
+                    color: Theme.palette.primaryColor1
+                    implicitWidth: 2
+                    implicitHeight: 24
+                    radius: 1
+                    visible: indicationText.cursorVisible
+                    SequentialAnimation on visible {
+                        loops: Animation.Infinite
+                        running: indicationText.cursorVisible
+                        PropertyAnimation { to: false; duration: 600; }
+                        PropertyAnimation { to: true; duration: 600; }
+                    }
+                }
+                onTextChanged: {
+                    if (previousText === text) {
+                        // Not sure why, but the textChanged event was triggered even if it didn't really
+                        return
+                    }
+                    if (root.maximumLength > 0) {
+                        if (text.length > root.maximumLength) {
+                            var cursor = cursorPosition;
+                            text = previousText;
+                            if (cursor > text.length) {
+                                cursorPosition = text.length;
+                            } else {
+                                cursorPosition = cursor - 1;
+                            }
+                        }
+                        previousText = text;
+                    }
+                }
+            }
+
+            StatusBaseText {
+                id: indicationText
+                anchors.bottom: parent.bottom
+                anchors.bottomMargin: Style.current.halfPadding
+                anchors.horizontalCenter: parent.horizontalCenter
+                font.pixelSize: 15
+                color: Theme.palette.baseColor1
+                text: qsTr("Write down %1 last %2 characters %3 of your private key").arg("<font color=\"" + Theme.palette.directColor1 + "\">").arg(root.maximumLength).arg("</font>")
```

Comment:
I meant keep the tags in, fill in the color using `arg()`

And now it needs a plural too since you have a number there :)

Reply:
done